### PR TITLE
Responsive Home Page

### LINKED ISSUE #3: make the frontend responsive
currently most HTML part of the website is not responsive and where it is the layout doesn't look so good.

### PR DESCRIPTION
**Pull request by tushhr**
Hey @MananJethwani, I have refactored the code and made it responsive. I have created some big changes as I have updated the Bootstrap Classes with the hard-coded CSS for flexibility.

### CODE REVIEW

**Comment by MananJethwani** [comment]
@tushhr from no on if you introduce any change to UI please try to attach a screenshot of the new page would be really helpful for me.

**Comment by tushhr** [comment]
Hey @MananJethwani, thanks for reminding me, and sorry for the inconvenience...
Home_1440
Home_768
Home_578
Home_420

**Review by MananJethwani — APPROVED** [review]

File: frontend/src/commponents/navBar.jsx [hunk 1]
```diff
@@ -45,33 +32,33 @@ function NavBar(props){
...
+                <>
+                  <button 
+                    onClick={props.onClose} 
+                    className="secButton"
+                  >
+                    Log in
+                  </button>
+
+                  <button
+                    onClick={props.onRegister}
```
Comment (MananJethwani) [inline frontend/src/commponents/navBar.jsx:51]: @tushhr this should be `props.onClose2`

**Review by MananJethwani — CHANGES_REQUESTED** [review]

File: frontend/src/commponents/navBar.jsx (hunk 1 above)
Comment (MananJethwani) [reply frontend/src/commponents/navBar.jsx:51]: also can you try to change `onclose` and `onclose2` to `toggleModal` and `toggleModal2`

Comment (tushhr) [reply frontend/src/commponents/navBar.jsx:51]: I will do that in the next PR, fixing the Signup bug and updating forms..

**Comment by tushhr** [comment]
@MananJethwani, I have fixed the margin error. Have a look.

Home_144

**Review by MananJethwani — APPROVED** [review]
thanks @tushhr

**Pull request merged**